int8[] data     # 二进制数据，长度可以使用 data.size() 查看